inc/templates/edit-form.tpl.php

```diff
@@ -134,3 +134,16 @@
 		<# } #>
 	</div>
 </script>
+
+<script type="text/html" id="tmpl-shortcode-ui-content">
+	<div class="field-block">
+		<div class="content-display-container">
+			<p class="content-display">{{ data.shortcode_content }}</p>
+			<a href="#" class="button-edit">Edit Content</a>
+		</div>
+		<div class="content-edit-container" style="display: none;">
+			<textarea class="content-edit">{{ data.shortcode_content }}</textarea>
+			<a href="#" class="button-save">Save Content</a>
```

Comment:
I think it makes sense to edit content, if the need be, directly from shortcake UI.